index.bs

```diff
@@ -289,7 +296,9 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     In the WebAuthn context, [=attestation=] is employed to <em>attest</em> to the <em>provenance</em> of an [=authenticator=]
     and the data it emits; including, for example: [=credential IDs=], [=credential key pairs=], signature counters, etc. An
     [=attestation statement=] is conveyed in an [=attestation object=] during [=registration=]. See also [[#sctn-attestation]]
-    and [Figure 3](#fig-attStructs).
+    and [Figure 3](#fig-attStructs). Whether or how the client platform conveys the [=attestation statement=] and [=AAGUID=] 
+    portions of the [=attestation object=] to the [=[RP]=] is described tby <dfn>attestation conveyance</dfn>, see 
```

Comment:
If you `<dfn>` "attestation conveyance" in #attestation-convey, you can remove the "see" clause.

Reply:
ok.